bigquery/google/cloud/bigquery/job.py

```diff
@@ -1469,6 +1463,12 @@ def from_api_repr(cls, resource):
     https://g.co/cloud/bigquery/docs/reference/rest/v2/jobs#configuration.query.writeDisposition
     """
 
+    table_definitions = _TypedApiResourceProperty(
+        'table_definitions', 'tableDefinitions', dict)
+    """See
```

Comment:
We should document that the dictionary expects its values to be ExternalConfig objects.

Reply:
Done.